server/models/conversation.js

```diff
@@ -0,0 +1,21 @@
+module.exports = (sequelize, DataTypes) => {
+    const Conversation = sequelize.define(
+        "conversation",
+        {
+            id: {
+                type: DataTypes.INTEGER,
+                primaryKey: true,
+                autoIncrement: true
+            }
+        }
+    );
+
+    Conversation.associate = function(models) {
+        Conversation.hasMany(models.Message);
+        Conversation.belongsTo(models.User, {as: "userOne"});
+        Conversation.belongsTo(models.User, {as: "userTwo"});
```

Comment:
Conversation could instead have many users

Reply:
ok I've updated it now, thanks !

Reply:
alright updated now to use a through table and put the read status on that table instead

Reply:
alright so I have a through table between users and conversations and also another for users and messages so that I can set read status on that for the messages.